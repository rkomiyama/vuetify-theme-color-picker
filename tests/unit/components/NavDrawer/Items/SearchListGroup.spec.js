import Vue from 'vue';
import Vuetify from 'vuetify';
import { shallowMount } from '@vue/test-utils'
import { createRenderer } from 'vue-server-renderer';
import SearchListGroup from '@/components/NavDrawer/Items/SearchListGroup.vue'

Vue.use(Vuetify)
let wrapper

describe('SearchListGroup.vue', () => {
  beforeEach(() => {
    wrapper = shallowMount(SearchListGroup)
  })

  afterEach(() => {
    wrapper.destroy()
  })

  it('is a Vue instance', () => {
    expect(wrapper.isVueInstance()).toBeTruthy();
  })

  it('SearchListGroup renders the same HTML', () => {
    const renderer = createRenderer();
    renderer.renderToString(wrapper.vm, (err, str) => {
    if (err) throw new Error(err);
      expect(str).toMatchSnapshot();
    });
  });
})
