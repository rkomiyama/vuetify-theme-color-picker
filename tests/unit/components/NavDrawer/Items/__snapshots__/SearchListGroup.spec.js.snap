// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchListGroup.vue SearchListGroup renders the same HTML 1`] = `
<v-list-group-stub color="primary" value="true" activeClass="" appendIcon="$vuetify.icons.expand" prependIcon="mdi-magnify" ripple="true">
  <v-list-item-group-stub value="0" activeClass="v-item--active" mandatory="true">
    <v-list-item-stub activeClass="" link="true" tag="div" class="pl-10">
      <v-list-item-action-stub>
        <v-icon-stub>mdi-account-search</v-icon-stub>
      </v-list-item-action-stub>
      <v-list-item-content-stub>
        <v-list-item-title-stub>Username</v-list-item-title-stub>
      </v-list-item-content-stub>
    </v-list-item-stub>
    <v-list-item-stub activeClass="" link="true" tag="div" class="pl-10">
      <v-list-item-action-stub>
        <v-icon-stub>mdi-face</v-icon-stub>
      </v-list-item-action-stub>
      <v-list-item-content-stub>
        <v-list-item-title-stub>Full name</v-list-item-title-stub>
      </v-list-item-content-stub>
    </v-list-item-stub>
  </v-list-item-group-stub>
</v-list-group-stub>
`;
